contribute/process/release-management.md

```diff
@@ -71,18 +70,17 @@ Each repository has the automation scripts setup to push the container images to
 Once a release made on a repository, Travis will trigger the release on the dependent repositories. The release tree looks as follows:
 
 - openebs/linux-utils
+  - openebs/dynamic-localpv-provisioner
   - openebs/jiva
+    - openebs/jiva-operator
   - openebs/cstor
   - openebs/libcstor
+    - openebs/m-exporter
     - openebs/istgt
       - openebs/cstor-operators
-      - openebs/external-storage
-        - openebs/maya
-          - openebs/velero-plugin
-          - openebs/cstor-csi
-          - openebs/upgrade
-          - openebs/jiva-operator
-            - openebs/jiva-csi
+        - openebs/velero-plugin
+        - openebs/cstor-csi
+        - openebs/upgrade
```

Comment:
q_: Is upgrade tagged by cstor-operators or cstor-csi?

Reply:
cstor-operators.